src/containers/BookList.js

```diff
@@ -0,0 +1,42 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+import { createBookAction, removeBookAction } from '../actions';
+
+const BookList = ({ books }) => {
+  const bookRows = books.map(book => (<Book key={`book-number-${book.bookId}`} book={book} />));
+
+  return (
+    <div>
+      <table>
+        <thead>
+          <th>
+            Book ID
+          </th>
+          <th>
+            Title
+          </th>
+          <th>
+            Category
+          </th>
+        </thead>
+        <tbody>
+          {bookRows}
+        </tbody>
+      </table>
+    </div>
+  );
+};
+
+BookList.propTypes = {
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
## Linter Disabled ⚠️ 
- [ ] There is an easy fix for this rule instead of the disable option. Let's lift this and improve on our propTypes definition, thank you!

Reply:
Fixed this